requirements.txt

```diff
@@ -1 +1,2 @@
+pytz
```

Comment:
Can you add this to `requirements/common.txt` instead? This `requirements.txt` file is just used for deploying websites - we shouldn't need to add anything here.